### PR TITLE
change default signature size to 384 tto support RSA-3072

### PR DESCRIPTION
**Pull request by ActoryOu**
change default signature size to 384 to support RSA-3072

Description
-----------

As mentioned in #238, change default signature size to 384 to support RSA-3072.

Checklist:
----------

- [x] I have tested my changes. No regression in existing tests.
- [x] My code is formatted using Uncrustify.
- [x] I have read and applied the rules stated in CONTRIBUTING.md. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: source/include/ota_private.h [hunk 1]
```diff
@@ -119,7 +119,7 @@
  * @brief A composite cryptographic signature structure able to hold our largest supported signature.
  */
 
-#define kOTA_MaxSignatureSize           256 /* Max bytes supported for a file signature (2048 bit RSA is 256 bytes). */
```
Comment (ravibhagavandas) [inline source/include/ota_private.h:122]: I think we should also change the struct name to match the change:

https://github.com/aws/ota-for-aws-iot-embedded-sdk/blob/1f2362f005f39a1847d645660ecaff93890cfcd7/source/include/ota_private.h#L372

Comment (ravibhagavandas) [inline source/include/ota_private.h:122]: I think we should also change the struct name to match the change:

https://github.com/aws/ota-for-aws-iot-embedded-sdk/blob/1f2362f005f39a1847d645660ecaff93890cfcd7/source/include/ota_private.h#L372

Comment (ActoryOu) [reply source/include/ota_private.h:122]: Fix in new commit.

Comment (ActoryOu) [reply source/include/ota_private.h:122]: Fixed in new commit.

**Review by ravibhagavandas — APPROVED** [review]

**Pull request merged**